staxing/assignment.py

```diff
@@ -389,7 +390,7 @@ def open_chapter_list(self, driver, chapter):
             By.XPATH,
             '//h2[contains(@data-chapter-section,"%s")]/a' % chapter
         )
-        if not bool(data_chapter.get_attribute('aria_expanded')):
+        if (data_chapter.get_attribute('aria-expanded')) == 'false':
```

Comment:
` == 'false'` or `true` is generally frowned upon in Python. It would be better to have an assistance function to check for a `bool` like:
`def is_true(self, value):
    check = str(value).lower()
    if check in ('yes', 'true', 't', '1'):
        return True
    elif check in ('no', 'false', 'f', '0'):
        return False
    raise`